document/core/valid/instructions.rst

```diff
@@ -154,6 +154,325 @@ Numeric Instructions
    }
 
 
+.. index:: simd instruction
+   pair: validation; instruction
+   single: abstract syntax; instruction
+
+.. _valid-instr-simd:
+
+SIMD Instructions
+~~~~~~~~~~~~~~~~~
+
+
+.. todo::
+   do we need v128.const here again? It's covered above by Numeric Instructions.
+
+.. todo::
+   can we define these instructions that begin with "v128." together with the numerics? The structure, "t.op" matches up. This will require us to specify in the conventions section of syntax/instructions to extend \binop, rather than define a new \vsbinop
```

Comment:
The separation into sections should follow the structure established in the syntax definition. In general, I think it's preferable to have SIMD stuff in its own section, since it's a rather separate feature, and quite large as well.

Reply:
Sg, I'll keep them separate, thanks!